test/extended/builds/completiondeadlineseconds.go

```diff
@@ -33,24 +31,20 @@ var _ = g.Describe("[builds][Slow] builds should have deadlines", func() {
 			err := oc.Run("create").Args("-f", sourceFixture).Execute()
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			g.By("starting the build with --wait flag")
-			out, err := oc.Run("start-build").Args("source-build", "--wait").Output()
-			fmt.Fprintf(g.GinkgoWriter, "\nstart-build output:\n%s\n", out)
-			o.Expect(err).To(o.HaveOccurred())
+			g.By("starting the source build with --wait flag and short timeout")
+			br, err := exutil.StartBuildAndWait(oc, "source-build", "--wait")
```

Comment:
same comment elsewhere.  we could only use the --wait flag if we're explicitly testing --wait behavior somewhere.